requirements-devel.txt

```diff
@@ -54,7 +54,7 @@ koji==1.26.1
     # via -r requirements.in
 mccabe==0.6.1
     # via flake8
-osbs-client @ git+https://github.com/containerbuildsystem/osbs-client@be934592d8c2a90c186bdab619a1dffb71cf62c1
+osbs-client @ git+https://github.com/containerbuildsystem/osbs-client@3572835efc3e2a7fbde414391d2bf6bf1108e915
```

Comment:
There seems to be already newer version 1b5823ec62be281cea8e096a034997ff86cdab86